lib/langchain/llm/openai.rb

```diff
@@ -17,7 +17,7 @@ class OpenAI < Base
       n: 1,
       temperature: 0.0,
       chat_completion_model_name: "gpt-3.5-turbo",
-      embeddings_model_name: "text-embedding-ada-002"
+      embeddings_model_name: "text-embedding-3-small"
```

Comment:
@emezac I modified the default model per this suggestion: https://github.com/andreibondarev/langchainrb/issues/506#issuecomment-1992709067

Reply:
Ok I agree